tests/utils.go

```diff
@@ -771,6 +773,52 @@ func BeforeTestSuitSetup() {
 	AdjustKubeVirtResource()
 }
 
+func disableKubectlCache() {
+	cachePaths := []string{
+		os.ExpandEnv("$HOME/.kube/cache/"),
```

Comment:
can we call `kubectl` and `oc` with the `--cache-dir` flag and point it to a temporary directory (where it does not have write permissions)? We run `functest` not inside a container, so if people run the testsuite locally, they will end up with changed permissions on their `.kube` folders.

Reply:
I am not sure if removing permissions on the dirs in general is a good thing (can we for instance leanup the dir if we are not root?). How about ousing `--cache-dir`, point it on every invocation to a new random directory and delete the directory after use?

Reply:
I thought we do run them in a container, so yea this is not a good idea.
I will modify it to use a temp directory

Reply:
Another option may also be to keep a unique  cache directory for the whole test-suite run and just delete it before every kubectl invocation.

Reply:
I thought about that, but because the `CreateCommandWithNS` function returns a `exec.Cmd`, we can't really tell when the command is actually invoked, only when it is instantiated (the issue here would be with parallel tests, which I'm not sure if we have right now).